src/providers/wallet/wallet.ts

```diff
@@ -1679,14 +1663,11 @@ export class WalletProvider {
     });
   }
 
-  public getProtocolHandler(coin: string, network?: string): string {
-    if (coin == 'bch') {
-      return network == 'testnet' ? 'bchtest' : 'bitcoincash';
-    } else if (coin == 'eth') {
-      return 'ethereum';
-    } else {
-      return 'bitcoin';
+  public getProtocolHandler(coin: Coin, network?: string): string {
+    if (coin == 'bch' && network === 'testnet') {
```

Comment:
maybe add a second optional parameter `network` to getProtocolPrefix, and add this exception inside getProtocolPrefix implementation? (just in case we need it from somewhere else)

Reply:
added second param: network